Update docs for `let mut` (#80)
grain-lang/grain-lang.org: src/constructs/boxes.md

title: Boxes
---

## On the subject of Boxes

> Boxes can be used for adding a mutable property to an immutable data structure, like a tuple or list, but `let mut` should be preferred for creating mutable value bindings.

Reviewer: I've been wanting to get some banners in here... This would be a great time to have an info banner for the second sentence.
```suggestion
Boxes can be used for adding a mutable property to an immutable data structure, like a tuple or a list. `let mut` should be preferred when creating mutable value bindings.
```

## Immutability of Boxes

> Just like any other binding in Grain, boxes are immutable. The box itself cannot be swapped out for another box (or anything else for that matter). However, the _contents_ of boxes can be changed.

Reviewer: ```suggestion
With `let` bindings in Grain, boxes are immutable. The box itself cannot be swapped out for another box (or anything else for that matter). However, the _contents_ of boxes can be changed.
```
